admin/app/javascript/spree/admin/controllers/variants_form_controller.js

```diff
@@ -41,6 +41,7 @@ export default class extends CheckboxSelectAll {
 
   connect() {
     super.connect()
+    this.locale = document.documentElement.lang || 'en'
```

Comment:
@caiodsc maybe it would be better to pass `I18n.locale` as a stimulus controller value? that way frontend locale will always match the backend one :)